### PR TITLE
Explain how critical OSS projects were selected

### PR DESCRIPTION
**Pull request by david-a-wheeler**
Let's explain the selection process in detail, since people
will ask questions about it. They shouldn't need to read
through meeting notes & watch meeting videos, just to figure it out :-).

Signed-off-by: David A. Wheeler <dwheeler@dwheeler.com>

### CODE REVIEW

**Comment by david-a-wheeler** [comment]
@xcorail @SecurityCRob @jnaulty - seem okay? I think we should at least provide *some* explanation.

File: README.md [hunk 1]
```diff
@@ -99,6 +97,61 @@ By using tools such as Multi-factor Authentication, the likelihood that such bad
...
+
+There are many ways to identify "critical" projects, so the
+Securing Critical Projects WG combined the results of several different
+analyses (each analyses are also called "Selection Criteria"),
+The WG then used human group review of this combined set of top candidates
+to create a final defensible list. The analysis ("selection criteria") for
+identifying candidate critical OSS projects included:
+
+* [OpenSSF Criticality Score](https://github.com/ossf/criticality_score): A top OpenSSF criticality score value. This metric prefers projects that are extremely active on specific forges. Such projects are likely to be important (at least to the participants). However, this is not a perfect measure; some projects will score low here and yet be very critical. Also, it currently only considers GitHub-hosted projects. As of 2021-11-23 the projects with the top scores are node, kubernetes, rust, and spark.
+* [Census Program II](https://www.coreinfrastructure.org/programs/census-program-ii/): Harvard preliminary analysis, uses SCA & dependency data. This tends to emphasize lower-level libraries that are depended on, tranisitively, by many.
```
Comment (xcorail) [inline README.md]:
```suggestion
* [Census Program II](https://www.coreinfrastructure.org/programs/census-program-ii/): Harvard preliminary analysis, uses SCA & dependency data. This tends to emphasize lower-level libraries that are depended on, transitively, by many.
```

**Review by xcorail — APPROVED** [review]
found a typo, otherwise LGTM

**Comment by david-a-wheeler** [comment]
Thanks!!

**Pull request merged**